src/Arcus.Templates.WebApi/Controllers/HealthController.cs

```diff
@@ -35,6 +38,10 @@ public HealthController(HealthCheckService healthCheckService)
         [HttpGet(Name = "Health_Get")]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status503ServiceUnavailable)]
+#if (ExcludeCorrelation == false && ExcludeCorrelation == false)
```

Comment:
This looks off, maybe use this instead?
```suggestion
#if (ExcludeOpenApi == false && ExcludeCorrelation == false)
```